.goreleaser.yml

```diff
@@ -0,0 +1,27 @@
+brews:
+  -
+    name: yor
+
+    tap:
+      owner: bridgecrewio
+      name: homebrew-tap
+      token: "{{ .Env.GITHUB_TOKEN }}"
+
+
+    url_template: "https://github.com/bridgecrewio/yor/releases/download/{{ .Env.GORELEASER_CURRENT_TAG }}/yor-{{ .Env.GORELEASER_CURRENT_TAG }}-darwin-amd64.tar.gz"
+
+    commit_author:
+      name: rotemavni
+      email: rotemavn@gmail.com
```

Comment:
Can we make it less personal here? Use github actions bot as a user, for example?